Lib/test/test_traceback.py

```diff
@@ -18,6 +18,7 @@
 import traceback
 from functools import partial
 
+MODULE_PREFIX = f"{__name__}." if __name__ == '__main__' else ''
```

Comment:
Any reason to use mixed quotes here?

Reply:
Copy pasting mostly! 😆 
Fixed, thanks for noticing.